newrelic/resource_newrelic_api_access_key.go

```diff
@@ -0,0 +1,317 @@
+package newrelic
+
+import (
+	"errors"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/newrelic/newrelic-client-go/pkg/apiaccesskeys"
+	"log"
+	"strings"
+)
+
+const (
+	KeyTypeIngest        = "INGEST"
```

Comment:
Maybe we see what Sander's PR is in, but perhaps we could move these constants to the client.

Reply:
You'll have enum-style types available from the client once my client PR is merged and released 🙂